resume.html

```diff
@@ -0,0 +1,154 @@
+                    <li><span class="bullet">Graduated with Distinction (U.K. equivalent of Honors)</span></li>
+                </ul>
+
+                <h4>BA in “BDIC: Animal Behavior”</h4>
+                <p><span>University of Massachusetts, </span> Amherst, MA</p>
+                <p>2004-2008</p>
+                <ul>
+                    <li><span class="bullet"><strong>Honors Thesis:</strong> “The effect of a feeding enrichment device on the behaviors of domestic dogs in animal shelters and its potential for welfare improvement.”</span></li>
+                    <li><span class="bullet">Member of IMPACT! Community Service Learning group and Commonwealth College Honors program.</span></li>
+                    <li><span class="bullet">Graduated summa cum laude</span></li>
+                </ul>
+            </section>
+
+            <section class="employment">
+                <h2>Employment</h2>
+                
+                <h4>Trinity Animal Hospital</h4>
+                <p class="job-title"><strong>Head of Training and Behavior</strong></p>
+                <p>September 2017 - April 2018</p>
+                <ul>
+                    <li><span class="bullet">Provided training and behavior consultations to clients.</span></li>
+                    <li><span class="bullet">Served as behavior consultant to veterinarians.</span></li>
+                    <li><span class="bullet">Created and taught training classes.</span></li>
+                    <li><span class="bullet">Trained veterinary clinic technical and kennel staff in low stress handling and restraint for dogs and cats.</span></li>
+                </ul>
+
+                <h4>North Carolina State University College of Veterinary Medicine</h4>
+                <p class="job-title"><strong>Clinical Behavior Technician</strong></p>
+                <p>January 2017 - August 2017</p>
+                <ul>
+                    <li><span class="bullet">Assist veterinarians in low-stress examinations for aggressive and/or fearful patients, performing restraint, muzzling, toweling, and venipuncture.</span></li>
+                    <li><span class="bullet">Create educational media for clients, veterinary students, and veterinary staff.</span></li> 
+                    <li><span class="bullet">Bi-weekly presentation on the veterinarian’s role in preventing behavior problems for fourth year students in clinical rotations.</span></li>
+                </ul>
+
+                <p class="job-title"><strong>Research Assistant</strong></p>
+                <p>March 2017-August 2017</p>
+                <ul>
+                    <li><span class="bullet">Collaborate with veterinary, computer science, and electrical engineering staff and students to design and carry out a pilot study crate training shelter dogs via a computer assisted training system.</span></li>
+                </ul>
+
+                <h4>Gentle Care Behavior & Training</h4>
```

Comment:
We didn't cover this much in class, but ampersands are a special character in HTML and so can't be put in there without doing something different. You have to write an _HTML entity_ instead. In this case, it would be `&amp;`.